doctr/models/detection/differentiable_binarization/base.py

```diff
@@ -150,10 +150,11 @@ def bitmap_to_boxes(
                     raise AssertionError("When assume straight pages is false a box is a (4, 2) array (polygon)")
                 _box[:, 0] /= width
                 _box[:, 1] /= height
-                boxes.append(_box)
+                # Add score to box as (0, score)
+                boxes.append(np.vstack([_box, np.array([0.0, score])]))
```

Comment:
What is 0 ?

Reply:
That's only a placeholder otherwise it wouldn't be possible to transport the objectness score here because with `assume_straight_pages=True` we have (N, 5) last is score and with `assume_straight_pages=False` we now have also (N, 5, 2) where score is [0.0, score]  up to the OCRPredictor where we detach it and go further with the real `loc_preds` (N, 4) / (N, 4, 2)

Reply:
Ah ok, so it's "useless" here

Reply:
correct ^^
see it as split between the loc_preds and the score :)